scripts/globals/mobskills/discharge.lua

```diff
@@ -0,0 +1,27 @@
+---------------------------------------------
+--  Discharge
+--  Description: Deals lightning damage. Additional effect: "Paralysis"
+--  Type: Magical (lightning)
+--  Utsusemi/Blink absorb: Wipes shadows
+--  Range: Aoe
+--  Notes:
+---------------------------------------------
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+---------------------------------------------
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffect = dsp.effect.PARALYSIS
+
+    MobStatusEffectMove(mob, target, typeEffect, 20, 0, 180)
```

Comment:
the typeEffect variable never gets used again and never changes, so why not just put the effect in directly instead of making a variable?

Reply:
Format was copied from another move but good point, i can change that.

Reply:
we're slowly cleaning up the old ones but we've a long way to go. Thanks.

Reply:
Everything should be changed